mousestyles/data/__init__.py

```diff
@@ -62,3 +62,80 @@ def load_all_features():
     other_features = [x.iloc[:, -1] for x in data_frames]
     other_features.insert(0, all_features_df)
     return pd.concat(other_features, axis=1)
+
+
+def load_intervals(feature):
+    """
+    Parameters
+    ----------
+    feature: {"AS", "F", "IS", "M_AS", "M_IS", "W"}
```

Comment:
> so this means you should run this function as `load_intervals("AS")`?
@shamindras That's my understanding.
@changsiyao I think a couple short examples in the docstring would make things really clear, when you get a chance. You can just make another section `Examples` and then add a couple [doctests](https://docs.python.org/3.5/library/doctest.html) as examples.